### ISSUE TITLE
Final Report Review - zz494

### ISSUE THREAD

**Issue by muggle-magic** [Johnzhong1468/MarketingCampaign#11]
This is a really interesting project about marketing. It has a wide range of applications. 

What I like this project:
- Visualization is great
- Great implementations of the tools we have learned in class
- This project has a wide range of applications in real world

What I am concerned about this project:
- My biggest concern is your criterion of measuring the goodness of your model. When you have an abnormally high accuracy on both training set and test set, I am concerned that your criterion is not good. Given that your interest is to predict whether people will buy the product because of phone calls, I assume from my common sense that "y" should equal to -1 for most of the time. Then the easiest way to classify is just to always predict y=-1. If 90% of your data have y=-1, this naive classification method will give you an "accuracy" of 0.90. I don't know the summary statistics of your data, but I think the distribution of y is definitely a topic that worths discussing. 
- Although the team has used a lot of methods, I would like to see more analysis of your results. For example, why Lasso perform badly whereas Ridge perform relatively good? What are the most influential features? Why do you think they are influential? You are currently working with data in the banking industry, can this result be generalized to other industry?

Conclusion:
Really interesting project with excellent implementations of the materials we have learned. But I am a little bit concerned about their abnormally high accuracy. Also I don't see much analysis.
